README.md

```diff
@@ -313,10 +315,20 @@ application for the first time, a typical use case is to perform an "initial syn
 Given create, update, delete are only sent on changes in the lifecycle of the
 entity, this extra event can be sent for all currently existing entities.
 
-
 `<url>` is the authoritative URL for the entity corresponding to the event
 (maximum 1024 characters, must use HTTPS scheme).
 
+`<timestamp>` (optional) is an integer number of milliseconds since the UNIX
+epoch and represents when the event occured. If unspecified it'll be set by the
+bus on reception.
+
+`<data>` (optional) is discouraged although not deprecated. It is intended when
+the RESN paradigm becomes impractical to implement — e.g. small, very
+frequently-changing representations that can't reasonably be fetched from the
+source and inconvenient to reify as changes in the domain (typically for storage
+reasons).
+
```

Comment:
Could you give us an example of when it would be appropriate to send data in an event?